test/merge/config/merge-config.test.js

```diff
@@ -15,4 +15,13 @@ describe('merge flag configuration', () => {
             done();
         });
     });
+    it('merges two configurations together with flag alias', (done) => {
+        const { stdout } = run(__dirname, ['--config', './1.js', '-m', './2.js'], false);
+        expect(stdout).toContain('merged.js');
+        stat(resolve(__dirname, './dist/merged.js'), (err, stats) => {
+            expect(err).toBe(null);
+            expect(stats.isFile()).toBe(true);
+            done();
+        });
+    });
```

Comment:
Let's override the other option, for example `filename`

Reply:
added https://github.com/webpack/webpack-cli/pull/1674/commits/1fdf2ae68039941fd47496bfc086fb5be77146a0 👍